src/platform/CatalogProvider.js

```diff
@@ -1,92 +0,0 @@
-// @flow
```

Comment:
Removing a file, if you confirm it's used in current LLD/LLM side is to be avoided, if you work on a v2 you must keep the file, have a console.warn that deprecates it if you want, but keep it until we port the LLD/LLM side.

otherwise it's a breaking changes and it must be avoided as much as possible.